tools/jenkins/run_performance.sh

```diff
@@ -37,4 +37,13 @@ BENCHMARKS_TO_RUN="bm_closure bm_cq bm_call_create bm_error bm_chttp2_hpack bm_c
 # Enter the gRPC repo root
 cd $(dirname $0)/../..
 
-tools/profiling/microbenchmarks/bm_diff.py -d origin/$ghprbTargetBranch -b $BENCHMARKS_TO_RUN
+AFFECTS_C_CPP=`python -c 'import sys; \
```

Comment:
I added `tools/jenkins/run_c_cpp_test.sh` and the idea is to have Jenkins run `tools/jenkins/run_c_cpp_test.sh tools/jenkins/run_performance.sh`. This should make it easy to extend the filtering to any future jobs specific to C/C++.